pkg/db/drivers/etcd/etcd.go

```diff
@@ -1494,12 +1510,12 @@ func (c *Client) getProfileByName(ctx *c.Context, name, storageType string) (*mo
 
 // GetDefaultProfile
 func (c *Client) GetDefaultProfile(ctx *c.Context) (*model.ProfileSpec, error) {
-	return c.getProfileByName(ctx, "default", "block")
+	return c.getProfileByNameAndType(ctx, "default", "block")
```

Comment:
it's better to use constants instead of magic string

Reply:
I have changed the magic string to constant string.